### DISCUSSION TITLE
Etherpad embeded hide (some of the) control bar

### DISCUSSION THREAD

**Discussion by slaudel** [ether/etherpad-lite#5835] [General]
Hello,

I'm embeding etherpad into a website, but I want to hide parts of or the whole control bar.
When calling a pad with "?showControls=false" this does not work. It is not working not only for our etherpad installation but also for others.
I tried some freely available etherpad instances but everytime the same as here: [https://etherpad.wikimedia.org/p/sjhfvjshbfis?showControls=false](https://etherpad.wikimedia.org/p/sjhfvjshbfis?showControls=false)
Maybe it is broken as it was in #2563? 

Thanks

**Comment by SamTV12345** [comment]
Thanks for raising that into awareness. Yes that feature seems broken. Unfortunately the code did change a lot. I'll need to track down why this is not working. The other query paremeter seem to work: https://etherpad.wikimedia.org/p/sjhfvjshbfis?showControls=false&showChat=false&showLineNumbers=false&useMonospaceFont=false
